### ISSUE TITLE
Any guidance in terms of reducing training instability in SSL

### ISSUE THREAD

**Issue by dongyaoli10x** [Pointcept/Pointcept#613]
Thank you for the repo and awesome work!

I have been training a PointTransformerV3 model similar to how Sonata trains in a different domain and there some some different loss functions etc.. One thing I noticed is that the training can be quite sensitive to the hyperparameters, such as learning rate, batch size, model structures (number of blocks, etc.). The sensitivity manifests in training stability: if the hyperparameters change seemingly slightly, the gradient norm can become very volatile at some epochs and then leads to very volatile loss curves. I am wondering if this something you guys have observed in Sonata and following SSL works. If you did, any suggestions on its possible origin and how to reduce it? Thanks again!

**Comment by Yujia-Zhang0913** [comment]
Yes, and this sensitivity in my experiments is mainly due to SPConv. In our current model, SPConv cannot be removed yet. But you can try some tricks for more stable pretraining:
- Change clip_grad to 1.0, which, in my experiments, will not influence the results much.
- Add eps=1e-4 to the AdamW optimizer, which will lower the results but be useful for more stable training.
